src/libsyncengine/syncpal/tmpblacklistmanager.cpp

```diff
@@ -148,7 +149,7 @@ bool TmpBlacklistManager::isTmpBlacklisted(ReplicaSide side, const SyncPath &pat
     return false;
 }
 
-void TmpBlacklistManager::insertInBlacklist(const NodeId &nodeId, ReplicaSide side) {
+void TmpBlacklistManager::insertInBlacklist(const NodeId &nodeId, const ReplicaSide side) {
```

Comment:
The argument `side` occupies the second position whereas it occupied the first position in `isTmpBlacklisted`.